profile/src/main/java/org/apache/karaf/profile/assembly/ConfigInstaller.java

```diff
@@ -38,9 +42,11 @@
 public class ConfigInstaller {
     private static final Logger LOGGER = LoggerFactory.getLogger(ConfigInstaller.class);
     private Path etcDirectory;
+    private Path homeDirectory;
     private List<String> pidsToExtract;
-    
-    public ConfigInstaller(Path etcDirectory, List<String> pidsToExtract) {
+
+    public ConfigInstaller(Path homeDirectory, Path etcDirectory, List<String> pidsToExtract) {
+        this.homeDirectory = homeDirectory;
```

Comment:
Same,  I don't think `homeDirectory` is required here.

Reply:
True, but baseDirectory is required.